compensation.adoc

```diff
@@ -2,79 +2,103 @@
+A good rule of thumb, if you are unsure about the value of your contribution: consider what you would charge for your work if you did it as a freelancer.
 
-== Vote on requests from others
+For example, it might be reasonable to request 50 BSQ for fixing some typos in a doc. But requesting 1000 BSQ for that same task, just because it took you a few hours to read through the doc, will probably be rejected.
+
+**For now, while the BSQ market is still young, we assume 1 BSQ = 1 USD**. This will change to a rate based on the market price of BSQ as the BSQ market matures.
 
-=== Review other contributors' requests
+See https://github.com/bisq-network/compensation/issues/277[this link^] for an example compensation request.
 
-Once the submission period is over, there is a short period to allow for contributors to review other contributors' requests and clear any uncertainties in the work done and value requested.
+=== File your compensation request in the DAO
 
-It's important to make use of this time so that you can make a well-informed vote on everyone's work.
+BSQ is issued on Bisq when a compensation request is approved through DAO voting, so your compensation request needs to be filed there in order for you to actually be paid.
 
-=== Vote
+Once you've documented all details of your request in a new issue on GitHub, make a new compensation request proposal on Bisq:
+
+.Make a new compensation request in the Bisq DAO.
+image::make-compensation-request.png[Make a new compensation request in the Bisq DAO]
+
+Make sure you select `Compensation request` as the proposal type. Also makre sure you use a name that stakeholders will recognize (or at least one they can cross-reference with your other online profiles like GitHub, Slack, forum, etc).
+
+When you're ready, click `Make proposal` to confirm your proposal for voting in the current cycle. **Proposal data cannot be edited**, so make sure everything is correct, especially the amount you're requesting.
+
+If you need to make a change, you can delete the proposal and make a new one while the proposal phase is still active, but you'll need to pay the proposal fee again. Once the proposal phase is over, proposals cannot be added or removed until the next proposal phase.
+
+NOTE: A compensation request is a type of proposal, and proposals cost a fee to make in the Bisq DAO. Currently that fee is 2 BSQ. Since a compensation request is intended to issue new BSQ, you'll also need to include a small amount of bitcoin to be colored into BSQ if your proposal is accepted—100 satoshis × [requested BSQ amount]. See more details in our https://docs.bisq.network/dao-user-reference.html#proposal-phase[User Guide].
+
+The BSQ and BTC is automatically added to your proposal transation by Bisq, but the BTC needs to be in your Bisq BTC wallet. Make sure you have enough in there before making your request. Keep in mind that you'll also need some BSQ and BTC for the other proposal phases (vote & reveal vote).
```

Comment:
vote reveal should not require BSQ/BTC?

Reply:
Vote reveal is a bitcoin transaction, so it requires BTC but not BSQ, I believe.

Reply:
Ah, correct. BTC mining fee only.